tests/e2e.sh

```diff
@@ -17,6 +17,8 @@
 # E2E tests for catalog functions.
 
 set -eo pipefail
-DIR="$(dirname "$0")"
+#DIR="$(dirname "$0")"
```

Comment:
what's going on with this file?

Reply:
We temporarily disable a test for contrib functions, since the test depends on the old kpt.
I will find some time to fix it in a followup PR.